### ISSUE TITLE
snapshots should detect when there snapshots with no corresponding test

### ISSUE THREAD

**Issue by stevenhuyn** [VineLang/vine#49]
Renaming a test will leave old snapshot files committed, `test.rs` should detect this.

Potentially we need a `cargo test` vs a `cargo test -- -update` as well.

**Comment by tjjfvi** [comment]
agreed, though detecting this is somewhat tricky given that someone can choose to run a subset of the tests

**Comment by stevenhuyn** [comment]
I was just imagining a struct that tracks `test_vi` and `test_vi_fail` calls
